src/Traits/HasAdditionalTransactionData.php

```diff
@@ -69,4 +71,28 @@ public function addAdditionalTransactionData($name, $value): self
         $this->additionalTransactionData->$name = $value;
         return $this;
     }
+
+    public function setShipping(?ShippingData $shipping): self
```

Comment:
Please make it clear what a variable contains. Shipping is not clear, could be a transaction too.
```suggestion
    public function setShipping(?ShippingData $shippingData): self
```